.github/sync/RELEASE.md

```diff
@@ -1,22 +1,23 @@
 <!-- These docs are updated and synced from https://github.com/conda/infra -->
 
 <!-- links -->
-[epic template]: {{ repo.url }}/issues/new?assignees=&labels=epic&template=epic.yml
+<!-- (some of these have been commented out for now, until the `repo.url` syntax works) -->
+<!-- [epic template]: {{ repo.url }}/issues/new?assignees=&labels=epic&template=epic.yml
+[compare]: {{ repo.url }}/compare
+[new release]: {{ repo.url }}/releases/new -->
```

Comment:
instead of commenting this out can we just put the full url here and avoid making all of the url changes inline?